.ci/lint

```diff
@@ -3,7 +3,7 @@ pipeline {
     options {
         timestamps()
         ansiColor('xterm')
-        timeout(time: 1, unit: 'HOURS') 
+        timeout(time: 3, unit: 'HOURS') 
```

Comment:
Just curious if you ever saw any go beyond an hour?

Reply:
Because its got a full lint option.  How long does a full lint take ?

Reply:
Gottcha, they didn't take that long although it may be a safe default. I need to ask the team here if they are wanting to do a full lint on merge forwards. I'd imagine they will.